pipeline/merger/src/main/scala/uk/ac/wellcome/platform/merger/rules/physicaldigital/SierraPhysicalDigitalPartitioner.scala

```diff
@@ -21,14 +17,20 @@ trait SierraPhysicalDigitalPartitioner extends WorkTagPartitioner {
       "sierra-system-number")
 
   private def isDigitalWork(work: UnidentifiedWork): Boolean =
-    work.data.workType match {
-      case None    => false
-      case Some(t) => t.id == "v" && t.label == "E-books"
+    work.data.items match {
+      case List(Unidentifiable(Item(List(_:DigitalLocation), _))) => true
+      case _ => false
+    }
+
+  private def isPhysicalWork(work: UnidentifiedWork): Boolean =
+    work.data.items match {
+      case List(Identifiable(Item(List(_:PhysicalLocation), _), _, _,_)) => true
+      case _ => false
```

Comment:
What happens in cases where there is more than one item? It seems like `isDigitalWork` and `isPhysicalWork` would both be `false`, is this by design?

Reply:
Yes. In the use case where we definitely want to merge we will have one item in each. This will make the merger more conservative but it is intentional, to avoid catching any of the other use cases where we might have a digitital item but we don't want to merge.

Reply:
That makes sense. It might be worth having a comment explaining this in the file, especially as the logic for this is split across both the partioner and the merger rule.